ReportUtilityBills/resources/util.rb

```diff
@@ -88,7 +89,7 @@ def self.simple(fuel_type, header, fuel_time_series, is_production, rate, bill,
       end
       monthly_fuel_cost[month] += hourly_fuel_cost[hour]
 
-      if fuel_type == FT::Elec && fuel_time_series.sum != 0 # has PV
+      if fuel_type == FT::Elec && sum_fuel_time_series != 0 # has PV
```

Comment:
Also for performance reasons, I moved the sum to outside the hourly loop. Between these two changes, the reporting measure runtime dropped from 1.0s to 0.15s.